src/gradualizer.erl

```diff
@@ -39,10 +39,13 @@
          type_check_forms/2
         ]).
 
--export_type([options/0]).
+-export_type([options/0, top/0]).
 
 -type options() :: proplists:proplist().
 
+%% This type is the top of the subtyping lattice.
+-type top() :: any().
```

Comment:
Could be -opaque or defined explicitly as a union of all types

Reply:
Thanks! Using opaque makes sense so I've used that. Defining it as a union of all types doesn't seem to be very practical, as I don't think that's possible without resorting to using `any()` in lots of places, which defeats the purpose.